chat-plugins/modlog.js

```diff
@@ -278,17 +302,26 @@ exports.commands = {
 		}
 
 		let addModlogLinks = Config.modloglink && (!hideIps || (targetRoom && targetRoom.isPrivate !== true));
-		// Let's check the number of lines to retrieve or if it's a word instead
 		let lines = 0;
-		if (!target.match(/[^0-9]/)) {
-			lines = parseInt(target || 20);
-			if (lines > RESULTS_MAX_LENGTH) lines = RESULTS_MAX_LENGTH;
+		if (target.includes('lines=')) { // undocumented line specification
+			const requestedLines = parseInt(target.split('lines=')[1].trim());
```

Comment:
I guess V8 does have fast slicing support, but still. This is less readable to me. I'd use `indexOf` and `slice`.